.github/workflows/callable-deploy.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Publish package to npm and create release on github'
+
+on:
+  workflow_call:
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GITHUB_EMAIL: test@email.com
+  GITHUB_USER: test-user
+  BRANCH: release
```

Comment:
This should be an input from the calling workflow

Reply:
Updated